resolver/SuggestPackage.cc

```diff
@@ -39,7 +39,8 @@ class PackageContext final {
     core::Context ctx;
     const core::packages::PackageInfo &currentPkg;
 
-    PackageContext(core::Context ctx) : ctx(ctx), currentPkg(ctx.state.packageDB().getPackageForFile(ctx, ctx.file)) {}
```

Comment:
i didn't check: can we delete `getPackageForFile`?

Reply:
Unfortunately not yet. It's still used to find the name that's cached on the `File`.

Reply:
oh  nvm, i see that it's still being used, but just to set the thing, so we can avoid calling it in weird places

could we make it private so that people don't call it?

Reply:
Good point, I'll have a look 👍

Reply:
I think it could be made to be private, but it would take some plumbing: there are some functions in the anonymous namespace of the packager that use it :/

Reply:
that's fine, np